luigi/scheduler.py

```diff
@@ -200,12 +200,45 @@ def _get_default(x, default):
         return default
 
 
+class OrderedSet(object):
```

Comment:
Is this copy-pasted from somewhere? If yes, then link to that place, if not, write it down explicitly as a comment.